src/components/AboutHome/AboutHome.js

```diff
@@ -49,6 +50,20 @@ const AboutHome = ({ user, setCurrentLink }) => {
     );
   }, []);
 
+  //Display login warnings for the login initial authorizer user with no responsibilities.
+  useEffect(() => {
+    //get the current user
+    const cdxUser = JSON.parse(localStorage.getItem("ecmps_user"));
+
+    //check if the user has initial authorizer role with no responsibilities
+    if (cdxUser?.roles?.includes(config.app.initialAuthorizerRole) && Object.keys(cdxUser?.facilities || {}).length === 0)
+    {
+      //display the warning message on the home page
+      const warningMessage = 'You have no facilities for which you are responsible. Please login to CBS to review and add facilities for which you are responsible.';
+      displayAppWarning(warningMessage);
```

Comment:
Should the user be logged out at this point?  We have already terminated the CDX session in AUTH API and removed the session. So, we need to remove the user information in local storage (take all steps required to sign the user out). Currently, it shows that the user is logged in the UI.  See handleSignOut() in easeyAuthApi.js

Reply:
Hi Yonatan. From what I remembered about the discussion with you and Mark this Monday, we talked about this case. and we expect that the user will be allowed to login to ECMPS and an informational warning should be rendered on top of the homepage. (however, for ticket #6344 add certificate statement check while login, any unsigned cert statements for the user will lead to a failed login) Please let me know if my understanding needs to be corrected. Thanks.

Reply:
Yep, you are correct.